test/models/order_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
```

Comment:
we are going to use rspec tests, not minitest, so it does not make any sense

Reply:
I guess you need to reconfigure your app not to generate minitests when you create new models/controllers etc.